sofiykam.md

```diff
@@ -0,0 +1,37 @@
+I will use the datasets to give the Jetson information to analyze (or use a camera to provide live videos/photos), 
+and finally continue training the Jetson and troubleshooting until it works more and more efficiently. I will 
+also need to practice running the program many times to prevent failures and incorrect measurements. In addition, the webcam 
+will be used to analyze the skin and predict what is happening to the skin of the patient, 
+and the pulse oximiter will be used to find data that can also be analyzed by the program. 
+
+## Budget
+
+| Product         | Supplier/Link                         | Cost   |
+| --------------- | ------------------------------------- | ------ |
+| Type C power supply   | https://www.amazon.com/dp/B08R7N4CP6/ref=sspa_dk_detail_3psc=1&pd_rd_i=B08R7N4CP6&pd_rd_w=MF4j6&contentid=amzn1.sym.dd2c6db76626-466d-bf04-9570e69a7df0&pf_rd_p=dd2c6db7-6626-466d-bf04-9570e69a7df0&pf_rd_r=N1XSQFM38F0KSV7X9CQ7&pd_rd_wg=x2yBP&pd_rd_r=27a3942d-ac16-4539-88205b34b0ac4265&s=pc&sp_csd=d2lkZ2V0TmFtZT1zcF9kZXRhaWxfdGhlbWF0aWM&spLa=ZW5jcnlwdGVkUXVhbGlmaWVyPUEyRjNTN1pENDc1UDRJJmVuY3J5cHRlZElkPUEwMTA3OTA5MjJOQ01QRUNTQk45RyZlbmNyeXB0ZWRBZElkPUEwMjg0MjM4MVY3TjJMNlA4V1NIQSZ3aWRnZXROYW1lPXNwX2RldGFpbF90aGVtYXRpYyZhY3Rpb249Y2xpY2tSZWRpcmVjdCZkb05vdExvZ0NsaWNrPXRydWU | $8.59  |
+| NVIDIA Jetson nano developer kit | https://www.amazon.com/NVIDIA-Jetson-Nano-Developer-945-13450-0000-100/dp/B084DSDDLT/ref=sr_1_2crid=35M189S5Q9LT3&keywords=nvidia+jetson+nano+developer+kit+2gb&qid=1672350344&sprefix=nvidia+jetson+nano+developer+kit+2gb%2Caps%2C133&sr=8-2  | $149.00 |
+| Ethernet Cable |https://www.amazon.com/AmazonBasics-RJ45-Cat-6-Ethernet-Patch-Cable-25-Feet-7-6-Meters/dp/B00N2VIWPY/ref=sr_1_3?crid=3G4N4R2JV0HAO&keywords=ethernet%2Bcable&qid=1672351370&sprefix=ethernet%2Bcable%2Caps%2C109&sr=8-3&th=1| $6.50|
+| Camera (Logitech C270 HD WEBCAM) | https://www.amazon.com/Logitech-Desktop-Widescreen-Calling-Recording/dp/B004FHO5Y6/ref=sr_1_1?crid=29DTQJIO6B0YV&keywords=Camera%2B(Logitech%2BC270%2BHD%2BWEBCAM)&qid=1672351499&sprefix=camera%2Blogitech%2Bc270%2Bhd%2Bwebcam%2B%2Caps%2C103&sr=8-1&th=1| $24.99|
+| Memory Card | https://www.amazon.com/Amazon-Basics-microSDXC-Memory-Adapter/dp/B08TJRVWV1/ref=sr_1_1_sspa?crid=FUSLKI6K20T7&keywords=memory%2Bcard&qid=1672351539&sprefix=memory%2Bcard%2Caps%2C113&sr=8-1-spons&spLa=ZW5jcnlwdGVkUXVhbGlmaWVyPUEzM1lOMklNUUxVRTBaJmVuY3J5cHRlZElkPUEwMjY2NDQxNFlBSU5VWllRQVhJJmVuY3J5cHRlZEFkSWQ9QTA2Njg3NDYzM1dDMk5KQ0QyMEJZJndpZGdldE5hbWU9c3BfYXRmJmFjdGlvbj1jbGlja1JlZGlyZWN0JmRvTm90TG9nQ2xpY2s9dHJ1ZQ&th=1| $13.77|
+|Wifi Card| https://www.amazon.com/Intel-Wireless-Ac-9260-2230-Gigabit/dp/B079QH5KW1/ref=sr_1_3?crid=2A4SQN5GIWXPM&keywords=wifi+card+intel+9260&qid=1672425982&sprefix=wifi+card+intel+9260%2Caps%2C125&sr=8-3| $13.35|
+|Pulse Oximeter| https://www.amazon.com/oximeter-Plethysmograph-Perfusion-portable-included/dp/B09XBC9MN3/ref=sr_1_24?keywords=pulse+oximeter+finger&qid=1672426825&sr=8-24| $19.99|
```

Comment:
is there a way for you to get raw data from this oximeter outside of their app and the display on the oximeter?

Reply:
I think so, I was originally thinking of just getting the data and inputting it into the program, but I would probably be able to connect the oximeter to the program and automatically send the measured data to the program to analyze.

Reply:
hmm... given that it's a closed-sourced product, it might be hard to do that? or would you be able to do some research and see how you would approach it? 

an alternate could be smth like this https://www.hackster.io/gatoninja236/open-source-pulse-oximeter-for-covid-19-4764c5?

Reply:
So I did some research and the way the Jetson gets the data is from a dataset that is downloaded and inputed - and it is possible to get information from the oximeter, make my own dataset, and do the same thing as with the CT scan and x-ray information and pictures. Later when it is trained I would be able to show (using the camera) the screen of the oximeter and it would tell me the results (it analyses pictures and x-ray/CT-scan data in the same way), or I could manually put it into the program on the computer.

Reply:
> So I did some research and the way the Jetson gets the data is from a dataset that is downloaded and inputed - and it is possible to get information from the oximeter, make my own dataset, and do the same thing as with the CT scan and x-ray information and pictures. Later when it is trained I would be able to show (using the camera) the screen of the oximeter and it would tell me the results (it analyses pictures and x-ray/CT-scan data in the same way), or I could manually put it into the program on the computer.

@sofiykam ahhh cool! i'm not sure whether manually inputting it into the program would be useful for each analysis but i bet you'll be able to figure it out :)